### PR TITLE
fix: minor typos

### PR DESCRIPTION
**Pull request by tudorpintea999**
Hello
I fixed some typos in the docs,
Hope it helps,
Br,
Tudor.

### CODE REVIEW

File: specs/derivation.md [hunk 1]
```diff
@@ -89,7 +89,7 @@
 > accommodate multiple such entities.
 
 [L2 chain derivation][g-derivation] — deriving L2 [blocks][g-block] from L1 data — is one of the main responsibility of
-the [rollup node][g-rollup-node], both in syncer mode, and in sequencer mode (where derivation acts as a sanity check
+the [rollup node][g-rollup-node], both in sync mode, and in sequencer mode (where derivation acts as a sanity check
```
Comment (0xHansLee) [inline specs/derivation.md:92]:
```suggestion
the [rollup node][g-rollup-node], both in validator mode, and in sequencer mode (where derivation acts as a sanity check
```

Comment (0xHansLee) [reply specs/derivation.md:92]: And also, please change the word `syncer` in L660 to `validator`.

File: specs/derivation.md [hunk 2]
```diff
@@ -294,7 +294,7 @@ Batcher transactions are encoded as `version_byte ++ rollup_payload` (where `++`
 | 0              | `frame ...` (one or more frames, concatenated) |
 
 Unknown versions make the batcher transaction invalid (it must be ignored by the rollup node).
-All frames in a batcher transaction must be parsable. If any one frame fails to parse, the all frames in the
+All frames in a batcher transaction must be parsable. If any one frame fails to parse, then all frames in the
```
Comment (0xHansLee) [inline specs/derivation.md:297]: For the same words with Optimism. 
https://github.com/ethereum-optimism/optimism/blob/b4c39a90f996b7131a9d03c7937e31e38445f22d/specs/derivation.md?plain=1#L310
```suggestion
All frames in a batcher transaction must be parsable. If any one frame fails to parse, the all frames in the
```

File: specs/exec-engine.md [hunk 3]
```diff
@@ -22,7 +22,7 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-This document outlines the modifications, configuration and usage of a L1 execution engine for L2.
+This document outlines the modifications, configuration and usage of an L1 execution engine for L2.
```
Comment (0xHansLee) [inline specs/exec-engine.md:25]: Same here. 
https://github.com/ethereum-optimism/optimism/blob/b4c39a90f996b7131a9d03c7937e31e38445f22d/specs/exec-engine.md?plain=1#L28
```suggestion
This document outlines the modifications, configuration and usage of a L1 execution engine for L2.
```

**Review by 0xHansLee — CHANGES_REQUESTED** [review]
Thank you for your contributions!

First of all, we are trying to minimize the differences from Optimism, so I suggested some changes for this.

**Comment by Pangssu** [comment]
Moved specifications to https://github.com/kroma-network/kroma-specs

**Pull request closed without merging**